pylibui/libui/box.py

```diff
@@ -54,13 +54,13 @@ def uiBoxDelete(box, index):
 # - int uiBoxPadded(uiBox *b);
 def uiBoxPadded(box):
     """
-    Sets the padding of the box.
+    Returns whether the box is padded.
 
     :param box: uiBox
     :return: int
     """
 
-    return clibui.uiBoxPadded(box)
+    return bool(clibui.uiBoxPadded(box))
```

Comment:
It this coercion necessary? Do we really want to return a boolean here?
I would suggest that everything inside pylibui/libui to be just a very thin wrapper around libui, and leave our pylibui python interface to pylibui/controls.